### ISSUE TITLE
Some packages could not be installed.

### ISSUE THREAD

**Issue by peterarends** [hieplpvip/ubuntu_canon_printer#13]
```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 grep : PreDepends: libpcre3 but it is not installable
 libgtk2.0-0:i386 : Depends: adwaita-icon-theme:i386 or
                             gnome-icon-theme:i386
                    Recommends: libgtk2.0-bin:i386
E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
```
For anyone getting these errors. I was not able to resolve them (in a timely matter), but commenting line 270 and 271 and running the script again made it work for me!

```
#apt-get -y install libatk1.0-0:i386 libcairo2:i386 libgtk2.0-0:i386 libpango1.0-0:i386 libstdc++6:i386 libpopt0:i386 libxml2:i386 libc6:i386
#check_error PACKAGE $?
```

**Comment by phannphong** [comment]
Commenting these lines let me finish the installing but when I access the shortcut I see the message "Printer Error" and "Check the DevicePath of /etc/ccpd.conf".
I've run script on Ubuntu 20.04.

**Comment by janos-r** [comment]
Ok, on the currently latest Mint21.2, this is what needs to be done:
1) You remove the `libgtk2.0-0:i386` dependency from the script. That is the one that creates the above mentioned libgtk2.0-bin:i386 problem. I don't know why exactly. And I don't know if skipping gtk2 looses on some features.
2) For some reason, the `service ccpd restart` part of the script fails, so you have to run it at the end manually.
3) Even though the script gets all the way to `Installation completed. Press any key to exit`, the .desktop file is not created on my desktop. I don't plan to debug it further, but this ending part of the script is definitely weird.

In summary, after removing the gtk2 dependency and running the ccpd restart manually, the `captstatusui -P LBP2900` command says "Ready to Print" and it really prints fine.
